Gemfile.lock

```diff
@@ -462,7 +462,7 @@ GEM
       connection_pool (~> 2.2)
     net-ldap (0.19.0)
     netrc (0.11.0)
-    newrelic_rpm (6.15.0)
+    newrelic_rpm (9.7.1)
```

Comment:
### [newrelic_rpm](https://newrelic.com/ruby)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 6.15.0 → 9.7.1

[_[change-log](https://github.com/newrelic/newrelic-ruby-agent/blob/main/CHANGELOG.md), [source-code](https://github.com/newrelic/newrelic-ruby-agent), [gem-diff](https://my.diffend.io/gems/newrelic_rpm/6.15.0/9.7.1)_]





<details>
<summary>Commits</summary>

A change of **4396** commits. See the full changes on [the compare page](https://github.com/newrelic/newrelic-ruby-agent/compare/6.15.0...9.7.1).

These are the first 10 commits:
- (740bd51) [Merge branch 'dev' into add_async_http_instrumentation](https://github.com/newrelic/newrelic-ruby-agent/commit/740bd519f892011f050bd44954c42ea9ac86e8f3)
- (8d0b0cb) [delete comments](https://github.com/newrelic/newrelic-ruby-agent/commit/8d0b0cbb014f8a24dbbcb59db9e72b74df4fcd2a)
- (ca6da27) [add changelog entry](https://github.com/newrelic/newrelic-ruby-agent/commit/ca6da2791a4aa94a73b91fd00ab2c027bf963514)
- (7eeee3f) [Ethon: focus on return_code for errors](https://github.com/newrelic/newrelic-ruby-agent/commit/7eeee3f0c99b09f63d4da4fce053e7fe6993113f)
- (30dc1fe) [Ethon instrumentation: default action, naming](https://github.com/newrelic/newrelic-ruby-agent/commit/30dc1fea0b3e98b62cfbf763f734aa3d839f2e1f)
- (55b25da) [Ethon: test request wrapper host determination](https://github.com/newrelic/newrelic-ruby-agent/commit/55b25da03937ec37972449309d9ef218a766996a)
- (b2481d0) [Merge pull request #2274 from newrelic/ethon_with_typhoeus](https://github.com/newrelic/newrelic-ruby-agent/commit/b2481d04ab509361580639496e509a2654601a81)
- (d947972) [Merge pull request #2271 from newrelic/rpm___used_to_sprinkl…](https://github.com/newrelic/newrelic-ruby-agent/commit/d9479728e09e44ae7f610d3caddc154d72bf01f1)
- (0b3edb6) [remove rescue from test](https://github.com/newrelic/newrelic-ruby-agent/commit/0b3edb6b65b7d18aede8cd6139c7e6754176205c)
- (00f0fce) [Update newrelic.yml](https://github.com/newrelic/newrelic-ruby-agent/commit/00f0fce2d0fe7f3105f743307c97a9f7d4a4b16f)


</details>